script.js

```diff
@@ -1,4 +1,109 @@
+// Global States
+var playerTurn = 1; // 1 = Player 1; 2 = Player 2
+var gamePhase = 1; // 1 = Dice Roll; 2 = Choose Order; 3 = Conclusion
+var playerRolls = [];
+// var player1Rolls = [];
+// var player2Rolls = [];
+var playerChoice = 0; // If 1, choose 1st numerical; If 2, choose 2nd numerical
+var finalNumber1 = 0; // Stores final number for Player 1
+var finalNumber2 = 0; // Stores final number for Player 2
+
+// Function: Returns random number from 1 to 6 -inclusive
+var rollDice = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNum = randomInteger + 1;
+  return diceNum;
+};
+
+// Function: Takes user input 1 or 2, to generate combined number's value with chosen 1st numerical
+var chosenOrderNum = function (choice, playerRolls) {
+  if (choice == 1) {
+    chosenNum = Number(String(playerRolls[0]) + String(playerRolls[1]));
```

Comment:
not sure why you had to convert a number to string before you converted it back to a number here